armi/settings/settingsIO.py

```diff
@@ -268,13 +270,16 @@ class Styles:
         """Enumeration of valid output styles"""
 
         short = "short"
+        medium = "medium"
         full = "full"
```

Comment:
I'm not really sure what the purpose of these three class variables is. My thought would be to delete them and just use the words `"short"`, `"medium"`, and `'long"` when they are needed. Less typing, no downside.

Reply:
yea I was just coding within the existing architecture, but you're right that it does end up causing more typing. @john-science do you have an opinion on me deleting this class?

Reply:
Perhaps, at least make them defined constant strings, so the spelling and capitalization is enforced:

https://github.com/terrapower/armi/blob/fe7f42f9ff838f5e3b8fb457ec5334d8ead05c1a/armi/physics/neutronics/settings.py#L29-L31

Passing around magic strings frequently causes trouble later. IMO